files/__addonLocation__/babel.config.json

```diff
@@ -1,9 +1,9 @@
 {
 <% if (typescript) { %>  "presets": [["@babel/preset-typescript"]],
-<% } %>  "plugins": [<% if (typescript) { %>
+<% } %>  "plugins": [
+    "@embroider/addon-dev/template-colocation-plugin",<% if (typescript) { %>
```

Comment:
I upstreamed the [interpolation code from `ember-codemod-v1-to-v2`](https://github.com/ijlee2/ember-codemod-v1-to-v2/blob/0.6.0/src/blueprints/ember-addon/__addonLocation__/babel.config.json). The expected output can be seen in these tests:

- [JavaScript project](https://github.com/ijlee2/ember-codemod-v1-to-v2/blob/0.6.0/tests/fixtures/new-v1-addon-javascript/output/new-v1-addon/babel.config.json)
- [TypeScript project](https://github.com/ijlee2/ember-codemod-v1-to-v2/blob/0.6.0/tests/fixtures/new-v1-addon-typescript/output/new-v1-addon/babel.config.json)